source/userguide/active_learning.rst

```diff
@@ -0,0 +1,269 @@
+Active Learning in MindMeld
+===========================
+
+Once a conversational application is deployed, capturing user logs can be highly beneficial to understand new trends and diversity in terms of how users are phrasing their queries for different intents. These logs can improve the quality of the training data and overall classifier performance. However, manually analyzing these logs and annotating them with the correct domain, intent and entity labels is costly and time consuming. Moreover, as the number of users for the system increase, the number of logs to be annotated also increases by many folds. This reduces the scalability of the manual annotation process. For such scenarios, we can use Active Learning.
```

Comment:
Once a conversational application is deployed, capturing user logs can be highly beneficial to understand new trends and diversity of how users are phrasing their queries for different intents. These logs can improve the quality of the training data and overall classifier performance. However, manually analyzing these logs and annotating them with the correct domain, intent and entity labels is costly and time consuming. Moreover, as the number of users for the system increase, the number of logs to be annotated also increases by many folds. This reduces the scalability of the manual annotation process. For such scenarios, we can use Active Learning.

Reply:
```suggestion
Once a conversational application is deployed, capturing user logs can be highly beneficial to understand new trends and diversity of how users are phrasing their queries for different intents. These logs can improve the quality of the training data and overall classifier performance. However, manually analyzing these logs and annotating them with the correct domain, intent and entity labels is costly and time consuming. Moreover, as the number of users of the system increase, the number of logs to be annotated also increases by many folds. This reduces the scalability of the manual annotation process. In such scenarios, we can use Active Learning.
```